src/javascripts/components/dashboard/rightPanel/editOptions.js

```diff
@@ -0,0 +1,44 @@
+import Vue from 'vue';
+import store from '../../../store.js';
+
+
+export default Vue.extend({
+  template: `
+    <div v-for="(index, option) in options" class="cc-rightPanel-edit-row">
+      <label style="display: none" :for="'cc-optionInput' + index">smthign dynamic</label>
+      <input
+        type="text"
+        :id="'cc-optionInput' + index"
+        class="cc-optionInput"
+        :value="options[index]"
+        @input="editOption"
+        @keyup.enter.prevent
+        autocomplete="off"
+      >
+      <img :src="srcForDeleteIcon" class="cc-rightPanel-garbageIcon">
+    </div>
+    <div class="cc-rightPanel-edit-row">
+      <button
+        class="cc-buttonReset cc-rightPanel-addOptionBtn"
+        @click.prevent
+      >Add Option</button>
+    </div>
+  `,
+  data() {
+    return {
+      options: ['a','b','c'],
+      srcForDeleteIcon: require('../../../../images/garbage.svg')
+    };
+  },
+  methods: {
+    addOption() {
+
+    },
+    editOption(event) {
+      this.$dispatch('updateOption', event.target.value);
+    },
+    deleteOption() {
```

Comment:
In general, don't add empty functions to a PR - my boss at IBM really drilled that one into my head..